packages/nextjs/services/plausible/index.ts

```diff
@@ -0,0 +1,33 @@
+const PLAUSIBLE_EVENT_ENDPOINT = "https://plausible.io/api/event";
+
+export async function trackPlausibleEvent(
+  eventName: string,
```

Comment:
I think we could do a ENUM / TYPE here instead of string. So it helps us to keep track of the events and be more type safe.

e.g.

```
export enum PlausibleEvent {
  JOIN_BG = "joinBG",
  SIGNUP_SRE = "signupSRE",
  CHALLENGE_SUBMISSION = "challengeSubmission",
}
```

and we use them as `PlausibleEvent.JOIN_BG`